swri_transform_util/package.xml

```diff
@@ -13,13 +13,12 @@
   <url>https://github.com/swri-robotics/marti_common</url>
   
   <buildtool_depend>catkin</buildtool_depend>
-  
+
   <depend>boost</depend>
   <depend>diagnostic_msgs</depend>
   <depend>geographic_msgs</depend>
   <depend>geometry_msgs</depend>
   <depend>libgeos++-dev</depend>
-  <depend>libopencv-dev</depend>
```

Comment:
Based on http://wiki.ros.org/opencv3 is seems that the key is opencv3, so I'll add that back.